Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/MSBuildForUnity.Common.props.template

```diff
@@ -1,11 +1,15 @@
 <Project>
   <PropertyGroup>
     <MSBuildForUnityVersion>0.8.0</MSBuildForUnityVersion>
+    
     <MSBuildForUnityGeneratedOutputDirectory><!--GENERATED_OUTPUT_DIRECTORY_TOKEN--></MSBuildForUnityGeneratedOutputDirectory>
+    <UnityProjectAssetsPath><!--UNITY_PROJECT_ASSETS_PATH_TOKEN--></UnityProjectAssetsPath>
+    <RecommendedBinaryOutputPath>$(UnityProjectAssetsPath)\Dependencies</RecommendedBinaryOutputPath>
```

Comment:
Seems like this should just be `RecommendedOutputPath` or `MSBuildForUnityDefaultOutputPath` since content will go there as well.

Reply:
I will take the second.